vscode-ext/src/extension.ts

```diff
@@ -420,21 +422,24 @@ class ConfigurationProvider implements vscode.DebugConfigurationProvider {
 		if (!await isTargetInFile()) {
 			let targets = await mirrordApi.listTargets(configPath);
 			if (targets.length === 0) {
-				await vscode.window.showErrorMessage(
+				vscode.window.showInformationMessage(
 					"No mirrord target available in the configured namespace. " +
-					"Set a different target namespace or kubeconfig in the mirrord configuration file.",
+					"You can run targetless, or set a different target namespace or kubeconfig in the mirrord configuration file.",
 				);
-
-				return undefined;
 			}
-
+			targets.push(TARGETLESS_TARGET_NAME);
 			let targetName = await vscode.window.showQuickPick(targets, { placeHolder: 'Select a target path to mirror' });
 
```

Comment:
Why not check here as well just like intellij that if the target list is empty and notify the user?